src/AutoMapper/XpressionMapper/XpressionMapperVisitor.cs

```diff
@@ -34,7 +34,7 @@ protected override Expression VisitParameter(ParameterExpression parameterExpres
         }
 
         protected override Expression VisitMember(MemberExpression node)
-        {
+        {
             if (node.NodeType == ExpressionType.Constant)

```

Comment:
Quite likely that code has no value.  The first curly brace below never gets hit stepping through the tests.

```
            if (node.NodeType != ExpressionType.MemberAccess)
            {
            }
```


 don't recall why I wrote it 2 years ago.